celery_pubsub/pubsub.py

```diff
@@ -84,6 +92,20 @@ def _topic_to_re(topic: str) -> re.Pattern[str]:
 _pubsub_manager: PubSubManager = PubSubManager()
 
 
+def subscribe_to(topic: str) -> typing.Callable:
+    def decorator(
+        func: typing.Callable[..., typing.Any]
+    ) -> typing.Callable[..., typing.Any]:
+        app_name, module_name = func.__module__.split(".", 1)
+        task_name = f"{app_name}.{module_name}.{func.__name__}"
```

Comment:
Better to use `__qualname__`. This will differentiate nested symbols (such methods in a class, or nested classes).